README.md

```diff
@@ -134,6 +134,12 @@ val keen = new Client {
 }
 ```
 
+### Scoped Keys
+
+`io.keen.client.scala.util.ScopedKeys` has a `encrypt` and `decrypt` method for handling
+[Scoped Keys](https://keen.io/docs/security/#scoped-key). If you have trouble with Java Exceptions see
+[this StackOverflow answer](http://stackoverflow.com/questions/6481627/java-security-illegal-key-size-or-default-parameters).
+
```

Comment:
Since scoped keys are a fairly involved topic to understand, an example might be good. The README on the Ruby gem does a nice job of being succinct about their purpose while linking for the full gory details. Here's a somewhat less succinct stab at it, see whatcha think:

> A utility object is provided for generating [Scoped Keys]. These are a must if you need to give out credentials for a client to access the Keen IO API directly, such as from client-side JavaScript. A scoped key has an associated set of query options that will always be applied when the key is used, so filters will enforce restrictions on what data the client can access.

> This example shows how you might generate per-user scoped keys in a backend Scala application.

```scala
import io.keen.client.scala.{ Client, Reader, Writer }
import io.keen.client.scala.util.ScopedKeys

// We'll assume you have a client instance, using your account-global API keys from configuration
val keen = new Client with Reader with Writer

val user = MyApp.authenticateAndLoadUser("credentialsFromRequest")
val securityOptions = s"""
  {
    "filters": [{
      "property_name": "user_id",
      "operator": "eq",
      "property_value": ${user.id}
    }],
    "allowed_operations": [ "read" ]
  }"""

// Create scoped key, encrypting with an account master key
val userReadKey = ScopedKeys.encrypt(keen.readKey, securityOptions)
```
> The resulting value can be rendered out to the browser during this user's login session for use with [Keen JS], for example. You can perform these steps per-request, or keys could be persisted once to your database and re-used.

> Of course you might instead scope on some other property like an organization or account ID, whatever is appropriate for data access control in your application.

One point I'm actually not certain of myself, though: can *any* account-global key (read, write, or master) be used for encrypting the scoped keys? The docs don't seem to specify this, the Ruby README suggests master.

[Scoped Keys]: https://keen.io/docs/security/#scoped-key
[Keen JS]: https://github.com/keen/keen-js